src/index.d.ts

```diff
@@ -984,6 +984,9 @@ export function TabNavigator(
   routeConfigMap: NavigationRouteConfigMap,
   drawConfig?: TabNavigatorConfig
 ): NavigationContainer;
+export function createAppContainer(
+  routeConfigMap: NavigationRouteConfigMap,
```

Comment:
Shouldn't it take a single component (of type `NavigationContainer`?). As far as I understand, `createAppContainer` cannot take a route config map.